test/simplifyExpression/arithmeticSearch/arithmeticSearch.test.js

```diff
@@ -6,11 +6,11 @@ function testArithmeticSearch(exprStr, outputStr) {
   TestUtil.testSimplification(arithmeticSearch, exprStr, outputStr);
 }
 
-describe.skip('evaluate arithmeticSearch', function () {
+describe('evaluate arithmeticSearch', function () {
   const tests = [
     ['2+2', '4'],
     ['2*3*5', '30'],
-    ['9/4', '9/4'], //  does not divide
+    ['9/4', '9 / 4'], //  does not divide
   ];
```

Comment:
I'm surprised this is all there is for tests.  It might be worth adding some additional tests.  In particular the code says that it prioritizes deep nodes, but the tests don't really show that.  It'd be good to have a nested expression and show the most deeply nested node gets evaluated.  Also, does only a single node get evaluated at a time?  There's lots of expressions this doesn't cover like unary minus, nthRoot, absolute value, and implicit multiplication.